requirements.txt

```diff
@@ -1,19 +1,25 @@
 app-common-python==0.2.3
+async-timeout==4.0.2
 attrs==20.3.0
 CacheControl==0.12.6
 certifi==2022.12.7
 chardet==3.0.4
+charset-normalizer==3.2.0
 confluent-kafka==1.5.0
 defusedxml==0.6.0
+exceptiongroup==1.1.2
```

Comment:
Don't know if these packages were added by mistake. Could you remove the newly added packages?